bench/workgen/workgen.cpp

```diff
@@ -1825,6 +1825,14 @@ ThreadRunner::op_run(Operation *op)
             _in_transaction = true;
         }
         if (op->is_table_op()) {
+            bool has_mirror = false;
+            {
+                const std::shared_lock lock(*_icontext->_dyn_mutex);
+                has_mirror = _icontext->_dyn_table_runtime.count(tint) > 0 &&
```

Comment:
why do we have `_icontext->_dyn_table_runtime.count(tint) > 0` as well, is `_icontext->_dyn_table_runtime[tint].has_mirror();` not sufficient to check if we have a mirror?

Reply:
As discussed offline, the operator `[]` creates a default value associated with the given key if the key does not exist. However I would like to make sure it exists and `count` can be used for that.